.github/CONTRIBUTING.md

```diff
@@ -13,6 +13,7 @@ We're so glad you asked!
    * [Contributing new katas](#contributing-new-katas)
    * [Style guide](#style-guide)
    * [Updating the Katas to the new QDK version](#updating-the-katas-to-the-new-qdk-version)
+   * [Updating the Quickref](#updating-the-quickref)
```

Comment:
Let's be a little formal here :-) I suspect not every contributor knows what "quickref" is, it's sort of hidden
```suggestion
   * [Updating the Q# Language Quick Reference](#updating-the-q-language-quick-reference)
```